spec/javascript/components/applications/ApplicationPage.test.js

```diff
@@ -1,22 +1,45 @@
 import React from 'react'
+import { clone } from 'lodash'
 import renderer from 'react-test-renderer'
-
 import ApplicationPage from 'components/applications/ApplicationPage'
 import sharedHooks from '../../support/sharedHooks'
 import domainApplication from '../../fixtures/domain_application'
 
-describe('ApplicationPage', () => {
-  sharedHooks.useFakeTimers()
+describe('ApplicationPage for app without flagged apps', () => {
```

Comment:
looks like the 2nd test here ends up being:
"ApplicationPage for app without flagged apps should render successfully with flagged apps table". I'm not sure that makes sense.

Should the "without flagged apps" be removed from this top-level describe?